lib/oci_utils/metadata.py

```diff
@@ -627,8 +572,8 @@ def refresh(self, get_public_ip=False):
         if metadata:
             self._metadata = OCIMetadata(metadata)
 
-        #_logger.debug('metadata after refresh [%s]' % str(self._metadata))
-        return result
+        return self
+
```

Comment:
no danger for read-consistency problems?

Reply:
We have in many solace use of locking and in many other nothing....  all this for mono-threaded programs. 
I've choose to remove locking and re-think about it if we need .